### PR TITLE
[v3-next] Update with main

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

update v3 next branch with changes on main

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [ ] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [ ] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [ ] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [ ] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [ ] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Comment by JillieBeanSim** [comment]
> Glad to see this branch being updated. We need to consider publishing the prereleases so that people start trying things out (even though we don't have any breakages yet 😋

Come July when we planned to release the first pre-release, we will have some breakages lol already working on some breaking changes that will affect extenders in #2161

File: packages/zowe-explorer-api/.eslintignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
+__mocks__/**
```
Comment (t1m0thyj) [inline packages/zowe-explorer-api/.eslintignore:1]: Lines 1 and 4 look like duplicates

File: packages/zowe-explorer/src/job/ZoweJobNode.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
  */
 
+// @ts-nocheck
```
Comment (t1m0thyj) [inline packages/zowe-explorer/src/job/ZoweJobNode.ts:12]: Curious why we use `@ts-nocheck` - feels like it could lead to undetected code smells?

**Review by t1m0thyj — APPROVED** [review]
Thanks for keeping the "next" branch up to date 👍

File: packages/zowe-explorer/src/job/ZoweJobNode.ts (hunk 2 above)
Comment (JillieBeanSim) [reply packages/zowe-explorer/src/job/ZoweJobNode.ts:12]: thanks @t1m0thyj these are things we can take care of either on main or as we refactor and add new to v3's next branch.

File: packages/zowe-explorer-api/.eslintignore (hunk 1 above)
Comment (zFernand0) [reply packages/zowe-explorer-api/.eslintignore:1]: Addressed in 18f1972ed92a4328ea933a6579ba15efe2665da6

File: packages/zowe-explorer/src/job/ZoweJobNode.ts (hunk 2 above)
Comment (zFernand0) [reply packages/zowe-explorer/src/job/ZoweJobNode.ts:12]: addressed in 18f1972ed92a4328ea933a6579ba15efe2665da6
Thanks @traeok

**Pull request merged**